src/main/java/cloud/fogbow/ras/core/plugins/interoperability/cloudstack/CloudStackErrorResponse.java

```diff
@@ -0,0 +1,24 @@
+package cloud.fogbow.ras.core.plugins.interoperability.cloudstack;
+
+import com.google.gson.annotations.SerializedName;
+import org.apache.http.client.HttpResponseException;
+
+public class CloudStackErrorResponse {
+
+    // TODO(chico) - Refactor: add this constants in the CloudStackConstants class
+    private static final String ERROR_CODE_KEY_JSON = "errorcode";
+    private static final String ERROR_TEXT_KEY_JSON = "errortext";
```

Comment:
Yes, it should be there. I didn't because I wasn't sure about doing a "commit" in the Common project. I created a new branch in the Common project and I am going to fix this in the RAS.

Reply:
Done